lametro/models.py

```diff
@@ -458,6 +459,32 @@ def upcoming_committee_meetings(cls):
         return meetings
 
 
+    @property
+    def board_event_minutes(self):
+        '''
+        This method returns the link to an Event's minutes. 
+
+        A small number of Events do not have minutes in 
+        a discoverable, corresponding EventDocument. 
+        For these, we can query board reports 
+        for indicative text, i.e., "minutes of the regular..."
+        '''
+        if 'regular board meeting' in self.name.lower():
+            try:
+                doc = self.documents.get(note__icontains='RBM Minutes')
+            except EventDocument.DoesNotExist:
+                try:
+                    date = self.start_time.date().strftime('%B %d, %Y')
+                    content = 'minutes of the regular board meeting held ' + date
+                    board_report = LAMetroBill.objects.get(ocr_full_text__icontains=content, bill_type='Minutes')
```

Comment:
L479-480 are new – we need to query `LAMetroBill` instead of `Bill`, because the `LAMetroBill` model has a custom manager that omits private bills. Previously, we were querying the `Bill` model, which led to links to private (unpublished) bills being displayed in the UI.